src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -21,6 +21,62 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
```

Comment:
The HVX contexts are configurable as 2 128 byte contexts or 4 64 byte contexts. A v60-hvx program can switch between these modes. So if a SW threads locks in 128 byte mode, then it means that no matter how many SW threads constitute the program, only one other thread can run hvx code (in 128 byte mode) or 2 other threads in 64 byte mode. Essentially, this way, the max number of threads is 4.